### ISSUE TITLE
Scrolling the sidebar scrolls the main page on iOS

### ISSUE THREAD

**Issue by leafsy** [ampproject/amphtml#24121]
## What's the issue?

Scrolling on the sidebar when it has reached the end would cause scrolling on the main page instead. Scrolling on the mask however does not have this issue. This seems to be a recurring bug that was reintroduced by newer version of iOS.

## How do we reproduce the issue?

1. Open [this example](https://amp-steven-master.firebaseapp.com) on an iOS device
3. Tap on the "toggle sidebar" button
2. Swipe up/down on the sidebar

## What browsers are affected?

Appears to affect all browsers on iOS devices.

## Which AMP version is affected?

Recent issue (1908211820140)

**Comment by wassgha** [comment]
Can confirm that this is resolved in iOS 13, probably not worth the effort to fix on previous iOS versions. Closing for now.
